spec/controllers/offline_reservations_controller_spec.rb

```diff
@@ -43,8 +43,9 @@
 
       it "logs the problem reservation" do
         post :create, params: params
-        log_event = LogEvent.find_by(loggable: reservation.order_detail, event_type: :problem_queue, metadata: {"cause"=>"new_offline_reservation"})
+        log_event = LogEvent.find_by(loggable: reservation.order_detail, event_type: :problem_queue)
         expect(log_event).to be_present
+        expect(log_event.metadata).to eq({"cause"=>"new_offline_reservation"})
```

Comment:
We can leave the curly braces off since they'd be implied within the parenthese.